pkg/daemon/ceph/client/pool.go

```diff
@@ -283,6 +283,20 @@ func setCommonPoolProperties(context *clusterd.Context, clusterInfo *ClusterInfo
 		}
 	}
 
+	// set max_bytes and max_objects quotas
+	if pool.Quotas.Enabled {
```

Comment:
If `max_bytes` and `max_objects` it's probably not worth calling those commands.

Reply:
To be sure, you are saying it would be better to not use the enabled setting?

Reply:
I'd suggest we don't need the `enabled` setting, we can just always set the quotas. If they aren't set in the pool CR, they will default to 0, which is the same behavior we already have, right?

Reply:
That is correct, assuming the pool didn't have a quota when the CR is applied (ie, set manually). Disabling quotas when they were set manually could possibly cause some issues. 

My thinking was that the `enabled ` setting would be used when the user explicitly wants Rook to handle the quotas.

Having said that, I'm ok with removing `enabled` as well. Please let me know which you prefer.

Reply:
That's a good point for backward compatibility sounds good to keep the `enabled: false` as the default, then we don't remove any quotas that might have been set separately.

Reply:
I'm still struggling with this a bit. Kubernetes doesn't use `enabled` as a field really, and I think that makes quite a bit of sense. What if we changed it around a bit so that we get the backward compatibility support we want while also not having `enabled`?

We can pretty easily make the `quotas` API spec nullable (i.e., a pointer), and then old specs that don't define quotas at all will just have a nil pointer to the quota spec, which implicitly means "disable control of quotas by Rook." When a user wants Rook to control quotas, that happens implicitly by setting `quotas` non-nil, even if the struct is empty. If the struct is empty, then Rook takes that to mean, disable quotas on the pool.

Reply:
Changes made. See conversation below.